src/quo2/foundations/colors.cljs

```diff
@@ -208,7 +208,7 @@
                60 "#1A4E52"}
    :pink      {50 "#F66F8F"
                60 "#C55972"}
-   :purple    {50 "#7140FD"
+   :purple    {50 "#8661C1"
```

Comment:
colors need to be verified with foundations - https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?type=design&node-id=619-9895&mode=dev